bundle/edu.gemini.spModel.core/src/main/scala/edu/gemini/spModel/core/Wavelength.scala

```diff
@@ -103,6 +70,10 @@ object Wavelength {
   implicit val WavelengthOrdering: scala.Ordering[Wavelength] =
     scala.Ordering.by(_.toNanometers)
 
-
+  /** @group Implicit Conversions */
+  implicit def fromLength(l: Length): Wavelength = {
```

Comment:
In theory this breaks a little bit with type safety because it allows people to pass an arbitrary `Length` when a `Wavelength` is needed but it allows for pretty code (see for example all the definitions in the test fixtures etc) and if someone really passes a light year as a wavelength then so be it. At least we still keep people from mixing `Wavelength`s with other things in the code.